frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -62,9 +70,22 @@ const SidebarComponent: React.FC<Props> = ({ schedule, major }) => {
   );
 };
 
+const SidebarComponent: React.FC<SidebarProps> = ({ schedule, major }) => {
+  const majorObj: Major | undefined = useSelector<AppState, Major | undefined>(
+    state =>
+      state.majorState.majors.find((majorObj: Major) => majorObj.name === major)
+  );
```

Comment:
Since `major` exists in the schedule state as a string and the sidebar requires a `Major` object, we need to find the corresponding `Major` object from Redux.

Reply:
Looks like we have a helper for this in `plan-helpers.ts` called `findMajorFromName`